pom.xml

```diff
@@ -51,8 +51,8 @@
   </properties>
 
   <scm>
-    <connection>scm:git:ssh://git@github.com/jenkinsci/kubernetes-credentials-provider-plugin.git</connection>
-    <developerConnection>scm:git:ssh://git@github.com/jenkinsci/kubernetes-credentials-provider-plugin.git</developerConnection>
+    <connection>scm:git:https://github.com/jenkinsci/kubernetes-credentials-provider-plugin.git</connection>
+    <developerConnection>scm:git:git@github.com:jenkinsci/kubernetes-credentials-provider-plugin.git</developerConnection>
```

Comment:
I don't know how these work in maven, but is this change right? I would expect `git@github.com/jenkinsci...`

Reply:
> is this change right?

Yes, this change is right.

Source: I am the world's leading expert in Jenkins SCM URLs in `pom.xml` files.

Reply:
👍

Reply:
it is a change without any difference.   `user@server:` is the shorter scp like syntax for `ssh://user@server/`

https://git-scm.com/book/en/v2/Git-on-the-Server-The-Protocols#_the_ssh_protocol